sdk/communication/azure-communication-administration/tests/test_phone_number_administration_client_async.py

```diff
@@ -0,0 +1,285 @@
+        assert items[0].phone_plan_id is not None
+
+    @AsyncCommunicationTestCase.await_prepared_test
+    @pytest.mark.live_test_only
+    async def test_get_phone_plan_location_options(self):
+        async with self._phone_number_administration_client:
+            location_options_response = await self._phone_number_administration_client.get_phone_plan_location_options(
+                country_code=self.country_code,
+                phone_plan_group_id=self.phone_plan_group_id,
+                phone_plan_id=self.phone_plan_id
+            )
+        assert location_options_response.location_options.label_id is not None
+
+    @AsyncCommunicationTestCase.await_prepared_test
+    @pytest.mark.live_test_only
+    async def test_get_release_by_id(self):
+        async with self._phone_number_administration_client:
+            phone_number_release_response = await self._phone_number_administration_client.get_release_by_id(
+                release_id=self.release_id
+            )
+        assert phone_number_release_response.release_id is not None
+
+    @AsyncCommunicationTestCase.await_prepared_test
+    @pytest.mark.live_test_only
+    async def test_list_all_releases(self):
+        async with self._phone_number_administration_client:
+            releases_response = self._phone_number_administration_client.list_all_releases()
+
+            items = []
+            async for item in releases_response:
+                items.append(item)
+        self.assertGreater(len(items), 0)
+        assert items[0].id is not None
+
+    @AsyncCommunicationTestCase.await_prepared_test
+    @pytest.mark.live_test_only
+    async def test_get_search_by_id(self):
+        async with self._phone_number_administration_client:
+            phone_number_search_response = await self._phone_number_administration_client.get_search_by_id(
+                search_id=self.search_id
+            )
+        assert phone_number_search_response.search_id is not None
+
+    @AsyncCommunicationTestCase.await_prepared_test
+    @pytest.mark.live_test_only
+    async def test_create_search(self):
+        searchOptions = CreateSearchOptions(
+            area_code=self.area_code_for_search,
+            description="testsearch20200014",
+            display_name="testsearch20200014",
+            phone_plan_ids=[self.phone_plan_id],
+            quantity=1
+        )
+        async with self._phone_number_administration_client:
+            search_response = await self._phone_number_administration_client.create_search(
+                body=searchOptions
+            )
+        assert search_response.search_id is not None
+
+    @AsyncCommunicationTestCase.await_prepared_test
+    @pytest.mark.live_test_only
+    async def test_cancel_search(self):
+        async with self._phone_number_administration_client:
+            await self._phone_number_administration_client.cancel_search(
+                search_id=self.search_id_to_cancel
+            )
```

Comment:
Same, is there a way to validate the result? (the search is successfully cancelled?)

Reply:
Added asserts to functions returning None